packages/bisect-summary/bisect-summary.0.3/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/lindig/bisect-summary/archive/0.2.zip"
```

Comment:
I am confused (even though you told me not to be). You have both `0.2` and `0.3` tagged in your repo but you want `0.3` in opam to actually be `0.2` on your repo? Isn't that going to confuse users?